Backpack/Dialog/Classes/BPKDialogIconView.m

```diff
@@ -34,12 +34,13 @@ + (CGSize)iconContainerSize;
 
 @implementation BPKDialogIconView
 
-- (instancetype)init {
+- (instancetype)initWithIconDefinition:(BPKDialogIconDefinition *)iconDefinition {
     CGSize size = [[self class] viewSize];
     self = [super initWithFrame:CGRectMake(0, 0, size.width, size.height)];
 
     if (self) {
         [self setupViews];
+        self.iconDefinition = iconDefinition;
```

Comment:
Wouldn't it be better to set this without using the setter i.e. the `ivar` before calling `setupViews` so that `setupViews` can rely on the value being present?

Reply:
Done